arch/arm/src/lc823450/lc823450_rtc.c

```diff
@@ -550,10 +548,8 @@ int up_rtc_settime(FAR const struct timespec *ts)
   up_rtc_set_default_datetime(tp);
 #endif /* CONFIG_RTC_SAVE_DEFAULT */
 
-#ifdef CONFIG_CLOCK_MONOTONIC
-  clock_gettime(CLOCK_MONOTONIC, &lastupdate_mono);
+  clock_systime_timespec(&lastupdate_mono);
```

Comment:
I think we should rename `lastupdate_mono` probably to `lastupdate_systime` since `clock_gettime(CLOCK_MONOTONIC,...)` is not used here anymore

Reply:
It's fine since systime also monotonically increase.